README.tfdoc.hcl

```
@@ -0,0 +1,826 @@
+          `apply` is considered complete.
+        END
+      }
+
+      variable "ordered_placement_strategy" {
+        type        = list(ordered_placement_strategy)
+        default     = []
+        description = <<-END
+          Service level strategy rules that are taken into consideration during
+          task placement. List from top to bottom in order of precedence.
+          Updates to this configuration will take effect next task deployment
+          unless `force_new_deployment` is enabled. The maximum number of
+          `ordered_placement_strategy` blocks is `5`. For details please
+          see [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service#ordered_placement_strategy).
+        END
+
+        attribute "type" {
+          required    = true
+          type        = string
+          description = <<-END
+            Type of placement strategy. Must be one of: `binpack`, `random`, or
+            `spread`.
+          END
+        }
+
+        attribute "field" {
+          type        = string
+          description = <<-END
+            For the `spread` placement strategy, valid values are `instanceId` (or
+            `host`, which has the same effect), or any platform or custom
+            attribute that is applied to a container instance. For the `binpack`
+            type, valid values are `memory` and `cpu`. For the `random` type, this
+            attribute is not needed. For more information, see
+            [Placement Strategy](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_PlacementStrategy.html).
+          END
+        }
+      }
+
+      variable "placement_constraints" {
+        type        = set(placement_constraint)
+        default     = []
+        description = <<-END
+          Rules that are taken into consideration during task placement. Updates
+          to this configuration will take effect next task deployment unless
+          `force_new_deployment` is enabled. Maximum number of
+          `placement_constraints` is `10`.
+        END
+
+        attribute "type" {
+          required    = true
+          type        = string
+          description = <<-END
+            Type of constraint. The only valid values at this time are
+            `memberOf` and `distinctInstance`.
+          END
+        }
+
+        attribute "expression" {
+          type        = string
+          description = <<-END
+            Cluster Query Language expression to apply to the constraint. Does
+            not need to be specified for the `distinctInstance` type. For more
+            information, see [Cluster Query Language](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-query-language.html)
+            in the Amazon EC2 Container Service Developer Guide.
+          END
+        }
+      }
+
+      variable "load_balancers" {
+        type        = set(load_balancer)
```

```suggestion
        type        = list(load_balancer)
```

this is a list afaik

---

And this:
```
"load_balancer": {
                                "nesting_mode": "set",
```